tools/dynamodbConnector.py

```diff
@@ -39,6 +34,8 @@ def insert_article(self, file_hash, url):
             )
         except ClientError as e:
             self.logger.error('Couldn\'t insert article [%s]', e)
+        except self.dynamodb.exceptions.ResourceNotFoundException:
```

Comment:
as an option in the future you can also import exceptions in the start of the file